src/App.tsx

```diff
@@ -23,6 +23,13 @@ function App() {
     setList([...list]);
   };
 
+  const updateTask = (selectedTodo: TodoItem, text: string) => {
+    if (text.trim() !== '') {
```

Comment:
what if we want to remove text from the item ?

Reply:
why would we need an empty todo item?